Topics/Software_Tools/PyAutoGUI/Automation_with_PyAutoGUI.md

```diff
@@ -166,7 +166,84 @@ pyautogui.doubleClick()
 ![sample_gif](./assets/move_folder.gif)
 
 ## Keyboard Controls
-Pending
+## Keyboard Controls
```

Comment:
It seems you have accidentally added another heading for the keyboard controls function.

Reply:
Yes, will remove the first heading.